docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md

```diff
@@ -0,0 +1,153 @@
+---
+aliases:
+  - ../data-sources/elasticsearch/
+  - ../features/datasources/elasticsearch/
+description: Guide for configuring the Elasticsearch data source in Grafana
+keywords:
+  - grafana
+  - elasticsearch
+  - guide
+  - data source
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Configure Elasticsearch
+title: Configure the Elasticsearch data source
```

Comment:
Is it possible to add "Grafana" to the title? That'll help SEO.

Reply:
I haven't added Grafana to any of the config docs...it's not part of the template I am creating but I can review that.  If I was doing a search on "configure Elasticsearch in Grafana" this page is the first one would come up, I think.

Reply:
All good. I got to chatting with Isabel earlier today, and we discussed the same thing wrt to viz docs but decided against it. If it shows up first, perfect. No need to consider any further! Thanks!